tests/org.eclipse.n4js.spec.tests/xpect-tests/Ch05_04__Redefinition_of_Members/Req70_OverrideFieldFinal.n4js.xt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2018 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+
+/* XPECT_SETUP org.eclipse.n4js.spec.tests.N4JSSpecTest END_SETUP */
+
+
+class A {
+	field: string;
+}
+interface I {
+	field2: number;
+}
+class B extends A implements I {
+	// XPECT errors --> "Cannot override non-final field A.field with a final field." at "field"
+	@Final @Override field: string = 'hello';
```

Comment:
At the moment, the @Final annotation for a field has two meaning:
- The field is read-only
- The field cannot be overridden in a subclass.

As discussed, we could also consider an alternative semantics of @Final:
- The field is read-only

But it is not part of this PR.